kpops/component_handlers/kafka_connect/connector_handler.py

```diff
@@ -210,27 +230,29 @@ def clean_connector(
         :param kwargs: Other values for the KafkaConnectResetter
         """
         suffix = "-clean"
-        release_name = connector_name + suffix
+        clean_up_release_name = connector_name + suffix
 
         log.info(
             magentaify(
                 f"Connector Cleanup: uninstalling cleanup job Helm release from previous runs for {connector_name}"
             )
         )
-        self.__delete_clean_up_job_release(release_name, suffix, dry_run)
+        self.__delete_clean_up_job_release(clean_up_release_name, suffix, dry_run)
 
         log.info(
             magentaify(
                 f"Connector Cleanup: deploy Connect {connector_type.value} resetter for {connector_name}"
             )
         )
-        self._helm_wrapper.helm_upgrade_install(
-            release_name=release_name,
-            suffix=suffix,
+
+        stdout = self._helm_wrapper.helm_upgrade_install(
+            release_name=trim_release_name(clean_up_release_name, suffix),
```

Comment:
Trimming also needs to be done when deleting the old job

Reply:
I am trimming it in the method:
```python
    def __delete_clean_up_job_release(
        self, release_name: str, suffix: str, dry_run: bool
    ) -> None:
        self._helm_wrapper.uninstall(
            namespace=self.namespace,
            release_name=trim_release_name(release_name, suffix),
            dry_run=dry_run,
        )
```

Reply:
I still need some tests for these trimmings. That is still WIP for this PR!

Reply:
Hm, I think it's safer to not trim in the method but extract the release name to a variable once and use that everywhere